packages/stark-core/src/logging.ts

```diff
@@ -2,4 +2,5 @@ export * from "./logging/actions";
 export * from "./logging/entities";
 export * from "./logging/reducers";
 export * from "./logging/services";
+export * from "./logging/testing";
```

Comment:
Actually, testing should not be exported here.
We added a testing entrypoint beside index.ts in the root of stark-core for exporting testing tools.
I think we should not get "MockStarkLoggingService" suggestion with autocompletion when we type StarkLogging. 
That's why it could be better to split them. 

The same for routing and session of course 😊

Reply:
I'll leave it as it is for the moment because that requires a big refactoring and a bit complicated to achieve.

I've created this issue to track this refactoring: #367